src/app/app.component.ts

```diff
@@ -46,7 +47,7 @@ export class AppComponent implements OnInit {
     } else if (this.showCookieAlertValue === 'false') {
       this.showCookieAlert = false;
     } else {
-      console.log('Error. New-user-cookie not found or initialized properly.');
+      AuthService.displayError('Error. New-user-cookie not found or initialized properly.');
```

Comment:
1. ...why is this being done in this PR?
2. Why can't you just initialize the cookie if it's missing?

Reply:
1. Because Anthony and I decided that our changes would fit with this branch. I understand that our changes are different though, which is weird for a PR. I think for now as long as the PR description is detailed enough then it's fine. Moving forward, we can get rid of this branch so we don't have multiple people making different changes like this on the same PR.

2. That's what line 40 is for. This else statement is if the code somehow skips past that point and there's no value received from the cookie (which would imply that it wasn't initialized or found).